client/state/selectors/can-upgrade-to-plan.js

```diff
@@ -0,0 +1,27 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { PLAN_FREE, PLAN_JETPACK_FREE } from 'lib/plans/constants';
+import { getCurrentPlan } from 'state/sites/plans/selectors';
+import { getPlan } from 'lib/plans';
+import { isJetpackSite } from 'state/sites/selectors';
+import { isSiteAutomatedTransfer } from 'state/selectors';
+
+export default function( state, siteId, planKey ) {
```

Comment:
Lets document this so we get a proper description in devdocs:

https://calypso.live/devdocs/selectors/canUpgradeToPlan?search=canupgrade&branch=update/can-upgrade-to-plan-selector